src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/Presentation/QuickInfoDisplayPanel.cs

```diff
@@ -1,77 +1,103 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
 using System.Text;
 using System.Windows;
 using System.Windows.Controls;
 using System.Windows.Documents;
 using System.Windows.Media;
+using Microsoft.CodeAnalysis.QuickInfo;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.QuickInfo
 {
+    internal class TextBlockElement
+    {
+        public string Kind { get; }
+        public TextBlock Block { get; }
```

Comment:
I'm not certain what direction this is.  Is this basically the owner of this "TextBlockElement", or is this something the TextBlockElement contains?